.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: ShellCheck
```

Comment:
I think won't need a ShellCheck linter in this repo.

Reply:
Sure.  The reason it's here is because everything is centrally managed now, and for now every repo gets the same common files: https://github.com/linux-system-roles/.github/pull/7
If it becomes a problem in the future to have this shellcheck where it isn't needed, then we can figure out how to create a "shellcheck_roles" group or otherwise exclude certain roles from getting the shellcheck action

Reply:
Thanks for the clarification. Regarding ShellCheck, we have an "in-house" GitHub Action that produces SARIF format and performs differential scans - [Differential ShellCheck GitHub Action](https://github.com/redhat-plumbers-in-action/differential-shellcheck)

Reply:
> Thanks for the clarification. Regarding ShellCheck, we have an "in-house" GitHub Action that produces SARIF format and performs differential scans - [Differential ShellCheck GitHub Action](https://github.com/redhat-plumbers-in-action/differential-shellcheck)

That's really cool!